test-packages/test-utils/api.d.ts

```diff
@@ -119,3 +124,17 @@ export function expectSuggestions(
   suggestions: UI5XMLViewCompletion[],
   expected: string[]
 ): void;
+
+export function readTestLibraryFile(
+  version: string,
+  fileName: string
+): Promise<FetchResponse>;
+
+export function downloadLibraries(version: TestModelVersion): Promise<void>;
+
+/**
+ * generateModel can take a long time in some cases now that it might download the libraries.
+ * Use this constant to set the timeout on the test/hook that calls it.
+ * Note: this.timeout(...) cannot be called from within an arrow function.
+ */
+export const GEN_MODEL_TIMEOUT = 3000;
```

Comment:
we may need even more to avoid random failures, e.g on slow containers in circle-ci
or slow developer machines.

Reply:
I could increase it to 5000